post.md

```diff
@@ -0,0 +1,226 @@
+# Robot Framework - using Request library for control RPI GPIO's
+
+Requests library is one of the most popular libraries implemented for Robot
+Framework. It is very important for testing scattered applications, so this is
+the first library I got to know in the Robot Framework. The library requires 
+installing `robotframework-requests` and `requests` for proper working. 
+`Collections` library is a standard built-in library.
+
+For checking `Request` library we can install Pi-GPIO-Server on Raspberry Pi.
+Server is able to control state of input and output of GPIOS using Rest API.
+
+Project is availiable on link: [Project weekend - GPIO Server](https://github.com/projectweekend/Pi-GPIO-Server)
+
+In the Rest Api we use methods:
```

Comment:
API and `we used following`

Reply:
@konopskyyyy in tags You wrote REST API. Capitalize it all and stick to one convention.